wfe/wfe.go

```diff
@@ -1512,3 +1534,18 @@ func (wfe *WebFrontEndImpl) deactivateRegistration(ctx context.Context, reg core
 		return
 	}
 }
+
+// addIssuingCertificateURLs() adds Issuing Certificate URLs (AIA) from a
+// X.509 certificate to the HTTP response. If the IssuingCertificateURL
+// in a certificate is not https://, it will be upgraded to https://
+func (wfe *WebFrontEndImpl) addIssuingCertificateURLs(response http.ResponseWriter, cert *x509.Certificate) error {
```

Comment:
Since the only part of `cert` you use is the `IssuingCertificateURL` you should just replace that with a `[]string` argument and only pass the part you need.